packages/core/src/client.ts

```diff
@@ -427,6 +400,39 @@ export class SkipRouter {
     return tx;
   }
 
+  async estimateGasForMessage(
```

Comment:
There is already a `getFeeForMessage` which I believe does the same thing

Reply:
The difference here is that `getFeeForMessage` creates a new stargateClient, while the `estimateGasForMessage` takes and reuses a stargate client from `executeCosmosMessage`. Im not sure but i thought that for each client creation the client will perform a `status` rpc call to establish the connection. 

In the latest commit (jst now) i also replaced the `getFeeForMessage` in `validateCosmosGasBalace`, which created the client twice; once in `validateGasBalances` in the messages for-loop and once in `getFeeForMessage`. 
I propose to keep `getFeeForMessage because it might be used externally(cause its public) but withing the client its now unused.